modules/storage/local.go

```diff
@@ -128,8 +128,16 @@ func (l *LocalStorage) URL(path, name string) (*url.URL, error) {
 }
 
 // IterateObjects iterates across the objects in the local storage
-func (l *LocalStorage) IterateObjects(fn func(path string, obj Object) error) error {
-	return filepath.WalkDir(l.dir, func(path string, d os.DirEntry, err error) error {
+func (l *LocalStorage) IterateObjects(prefix string, fn func(path string, obj Object) error) error {
+	dir := l.dir
+	if prefix != "" {
+		prefix_trim := filepath.Clean(prefix)
+		dir = filepath.Join(l.dir, prefix_trim)
+		if !strings.HasPrefix(dir, strings.TrimSuffix(l.dir, "/")) {
```

Comment:
Looks like it's useless except you use `filepath.Clean` on the dir

Reply:
Yeah. filepath.Join seems shorten result path as Clean method called.